src/renderers/dom/shared/DOMPropertyOperations.js

```diff
@@ -174,8 +174,10 @@ var DOMPropertyOperations = {
         var propName = propertyInfo.propertyName;
         // Must explicitly cast values for HAS_SIDE_EFFECTS-properties to the
         // property type before comparing; only `value` does and is string.
+        // Must set `value` property if it is not null and not yet set.
```

Comment:
@zpao It's a question of being either clear or general. In any case, I'm not aware of other attributes which may need to be handled similarly. And currently it is the only one. The previous comment line mentions this, so I believe writing `value` makes it more clear for others reading this code.